rfcs/packaging/communicating-convergence-status.md

```diff
@@ -0,0 +1,142 @@
+
+Notes:
+
+- Charting and maybe experiments would probably fall into the "preview (legacy)" category
+- Suite packages should list which components are converged or not
+
+## Discarded Solutions
+
+<!-- As you enumerate possible solutions, try to keep track of the discarded ones. This should include why we discarded the solution. -->
+
+## Open Issues
+
+<!-- Optional section, but useful for first drafts. Use this section to track open issues on unanswered questions regarding the design or proposal.  -->
+
+### Should we change anything about `react-cards`?
+
+The Card exported by default from `react-cards` is not converged and was never quite officially released, and there's work on a converged version in `src/next`. I'm not sure how many people were using the old Card.
+
+Should we move the old implementation elsewhere (to experiments maybe?) or delete it so the new one can be exported by default? If we do that, what happens to existing consumers?
+
+Another sneaky way out would be moving the new card to a new package `react-card` (singular), which is a more standard name but could definitely lead to confusion having both.
+
+### Should `@fluentui/react-*` mean converged? (remove `react-` prefix from old packages?)
+
+For communication purposes to consumers, I'm not sure that renaming all our React-based packages to have the `react-` prefix (which we're using for converged packages) was the right thing to do, since it worsens confusion about which packages are converged.
+
+While the status badges proposed above would help clarify this, it might also be worth reconsidering some of the new package names before we release version 8 and are stuck with the names for awhile.
+
+Some packages with this issue:
+
+| Package                    | Old name           | Notes                                                        |
+| -------------------------- | ------------------ | ------------------------------------------------------------ |
+| `react-cards`              | same               | see above                                                    |
+| `react-charting`           | `charting`         | no plan to converge                                          |
+| `react-date-time`          | `date-time`        | exported from FUIR v8                                        |
+| `react-docsite-components` | `example-app-base` | legacy doc site stuff, no plan to converge                   |
+| `react-experiments`        | `experiments`      | random mix of active/dead                                    |
+| `react-file-type-icons`    | `file-type-icons`  |                                                              |
+| `react-monaco-editor`      | `tsx-editor`       | react wrapper for monaco editor, probably won't converge     |
+| `react-theme`              | `theme`            | utility package which I think isn't involved in convergence? |
```

Comment:
`react-theme` is a package that currently contains the converged theme design tokens available from figma. I'm not entirely sure what is in `theme` since the token names and colors are different.

Whether it will stay is still questionable pending the outcome of the token pipeline. It's possible that there might be some figma -> TS autogeneration that spits out npm theme packages

Reply:
Oh sorry I mixed it up with something else. I'll remove that from the list.